docs/Installation-Windows.md

```diff
@@ -0,0 +1,190 @@
+# Installing ML-Agents for Windows
+
+ML-Agents supports Windows 10. While it might be possible to run ML-Agents using other versions of Windows, we only test Windows 10. (We also do not test running Windows on a VM like Bootcamp or Parallels).
```

Comment:
"we only test Windows 10" --> "it has not been tested on other versions"
"(We also do not test running Windows on a VM like Bootcamp or Parallels)." --> "Furthermore, ML-Agents has not been tested on a Windows VM such as Bootcamp or Parallels.

Reply:
done